GiniCapture/Classes/Core/GiniConfiguration.swift

```diff
@@ -623,6 +623,16 @@ import UIKit
      */
     @objc public var shouldShowDragAndDropTutorial = true
     
+    /**
+     Sets if the custom help menu items should be added on the help menu screen
+     */
+    @objc public var hasCustomMenuItems = false
```

Comment:
We could infer if it has custom items by checking if `customMenuItems` is not empty. And then clients don't need to set this flag in addition to setting the custom items.